validator/src/main/java/com/amazon/aoc/validators/ContainerInsightStructuredLogValidator.java

```diff
@@ -46,8 +47,12 @@ void init(Context context, FileConfig expectedDataTemplate) throws Exception {
       FileConfig fileConfig = new LocalPathExpectedTemplate(FilenameUtils.concat(
           expectedDataTemplate.getPath().toString(),
           logType + ".json"));
-      String templateInput = mustacheHelper.render(fileConfig, context);
-      schemasToValidate.put(logType, parseJsonSchema(templateInput));
+      try {
+        String templateInput = mustacheHelper.render(fileConfig, context);
+        schemasToValidate.put(logType, parseJsonSchema(templateInput));
+      } catch (IOException e) {
+        log.info("The " + logType + " was not found for this expected template path.");
```

Comment:
Maybe make this debug instead of info. Typically will not need to see this.

Reply:
Will do